modules/_labs/button/react/lib/Button.tsx

```diff
@@ -0,0 +1,206 @@
+import * as React from 'react';
+import {colors} from '@workday/canvas-kit-react-core';
+import {focusRing, GrowthBehavior} from '@workday/canvas-kit-react-common';
+import {CanvasSystemIcon} from '@workday/design-assets-types';
+import {ButtonVariant, ButtonSize, ButtonColors, DropdownButtonVariant} from './types';
+import {ButtonContainer, ButtonLabel, ButtonLabelData, ButtonLabelIcon} from './parts';
+
+export interface ButtonProps extends React.HTMLAttributes<HTMLButtonElement>, GrowthBehavior {
+  /**
+   * The variant of the Button.
+   * @default ButtonVariant.Secondary
+   */
+  variant: ButtonVariant;
+  /**
+   * The size of the Button.
+   * @default ButtonSize.Medium
+   */
+  size: ButtonSize;
+  /**
+   * The ref to the button that the styled component renders.
+   */
+  buttonRef?: React.Ref<HTMLButtonElement>;
```

Comment:
Do we want this to remain `buttonRef`? or change to `ref`? I think this is a remnant from Emotion 9.

Reply:
I don't think so, this is just what we name our refs right? `<component>Ref` is the naming convention for the prop API. This demonstrates what type of element the ref is referring to by name without relying strictly on documentation to indicate what it's a ref for.

Or are you alluding to the `innerRef` to `ref` change in the styled component interface in Emotion 10?

Reply:
Ah I guess that's true. I mentioned Emotion 9 because I think we originally didn't want to expose Emotions API by using `innerRef`.

Will leave as is 👍